packages/core/src/utilities/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './config/networkConfigs'
```

Comment:
lets rename networkConfigs to index.ts so that we can import it here just as ./config